coupon/src/main/resources/properties/datasource.yml

```
@@ -17,8 +17,8 @@ spring:
   sql:
     init:
       mode: always
-      schema-locations: classpath:db/schema.sql
-      #data-locations: classpath:db/data.sql # db 시작할때 실행시킬 script (더미 데이터 생성)
+      #schema-locations: classpath:db/schema.sql
```

YAMLlint (1.35.1)

[warning] missing starting space in comment

(comments)